CI: don't pass python runtime to invoke tasks

chouquette opened the pull request:
<!--
* Contributors are encouraged to read our [CONTRIBUTING](/CONTRIBUTING.md) documentation.
* Both Contributor and Reviewer Checklists are available at https://datadoghq.dev/datadog-agent/guidelines/contributing/#pull-requests.
* The pull request:
  * Should only fix one issue or add one feature at a time.
  * Must update the test suite for the relevant functionality.
  * Should pass all status checks before being reviewed or merged.
* Commit titles should be prefixed with general area of pull request's change.
* Please fill the below sections if possible with relevant information or links.
-->
### What does this PR do?

Remove `--python-runtimes` parameters from the CI jobs.

This is split out of https://github.com/DataDog/datadog-agent/pull/30035 to ease reviews

### Motivation

It's now always set to 3 which is the default, so we don't need to pass it.

### Describe how to test/QA your changes

### Possible Drawbacks / Trade-offs

### Additional Notes
<!--
* Anything else we should know when reviewing?
* Include benchmarking information here whenever possible.
* Include info about alternatives that were considered and why the proposed
  version was chosen.
-->

chouquette committed:
CI: don't pass python runtime to invoke tasks

It's now always set to 3

FlorentClarret reviewed (approved):
You can also drop this one I think: https://github.com/DataDog/datadog-agent/blob/9d1169042aaeb82bdd14e049a399220592765600/tasks/libs/common/omnibus.py#L144

chouquette commented:
> You can also drop this one I think:
> 
> https://github.com/DataDog/datadog-agent/blob/9d1169042aaeb82bdd14e049a399220592765600/tasks/libs/common/omnibus.py#L144

Indeed, I'll make the change in the main PR and will cherry pick the updated commit as I split out the changes in `tasks`!
Thanks

chouetz commented on .gitlab/integration_test/windows.yml:
```diff
@@ -26,7 +26,6 @@
       -e CI_PIPELINE_ID="${CI_PIPELINE_ID}"
       -e CI_PROJECT_NAME="${CI_PROJECT_NAME}"
       -e AWS_NETWORKING=true
-      -e PY_RUNTIMES="$PYTHON_RUNTIMES"
```
As a consequence, I suppose we can also clean occurrences of `PY_RUNTIMES`. I was thinking of the `winbuildscripts` and [it seems](https://github.com/search?q=repo%3ADataDog%2Fdatadog-agent%20PY_RUNTIMES&type=code) there is also some possible cleanup in omnibus.

chouquette replied:
This is handled by another commit in the original PR: https://github.com/DataDog/datadog-agent/pull/30035/commits/2b76319b42afce04e21c731b03c5a9ebe3f33b77

chouquette committed:
reinstate python runtime environment variables

They are still used by some scripts and will be dropped again in a later
commit

chouquette commented:
/merge

f4usto commented on .gitlab/package_build/installer.yml:
```diff
@@ -21,7 +21,7 @@
     # NOTE: for now, we consider "ociru" to be a "redhat_target" in omnibus/lib/ostools.rb
     # if we ever start building on a different platform, that might need to change
-    - inv -e omnibus.build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR  ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod" --system-probe-bin=/tmp/system-probe --host-distribution=ociru --install-directory="$INSTALL_DIR"
+    - inv -e omnibus.build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --base-dir $OMNIBUS_BASE_DIR  ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod" --system-probe-bin=/tmp/system-probe --host-distribution=ociru --install-directory="$INSTALL_DIR"
```
So the installer will also be branched into a v6?

chouquette replied:
I don't think that's planned. This was mostly there because it was based on an existing config which allowed targeting python2
